src/components/View.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const View = (props) => {
+    let out = '';
+    if(!props.view){
+        out= '';
+    } else {
+        out = <img className="pred-view" src={props.view} alt="url is not correct" style={{transform: `rotate(${props.flip}deg)`}}/>;
+    }
```

Comment:
if (props.view) {
out = <img className="pred-view" src={props.view} alt="url is not correct" style={{transform: `rotate(${props.flip}deg)`}}/>;
}

!props.view - reduntant condition